metals/src/main/scala/scala/meta/internal/metals/LanguageClientLogger.scala

```diff
@@ -4,14 +4,19 @@ import org.eclipse.lsp4j.MessageParams
 import org.eclipse.lsp4j.MessageType
 import scribe.LogRecord
 import scribe.output.LogOutput
+import scribe.output.format.OutputFormat
 import scribe.writer.Writer
 
 /**
  * Scribe logging handler that forwards logging messages to the LSP editor client.
  */
 object LanguageClientLogger extends Writer {
   var languageClient: Option[MetalsLanguageClient] = None
-  override def write[M](record: LogRecord[M], output: LogOutput): Unit = {
+  override def write[M](
+      record: LogRecord[M],
+      output: LogOutput,
+      outputFormat: OutputFormat
```

Comment:
Small change needed here. The signature for `write` has changed to need `outputFormat`.